src/main/java/carRace/domain/Referee/Referee.java

```diff
@@ -1,30 +1,28 @@
 package carRace.domain.Referee;
 
 import carRace.domain.car.Car;
-import carRace.domain.car.CarGroups;
+import carRace.domain.car.CarGroup;
 import carRace.domain.car.CarName;
 import carRace.domain.car.CarNames;
-import carRace.domain.car.MoveDistance;
 import java.util.List;
 import java.util.NoSuchElementException;
 import java.util.stream.Collectors;
 
 public class Referee {
 
-    public CarNames findWinners(final CarGroups cars) {
-        return findCarNameWithWinningScore(cars, findWinningScore(cars));
+    public CarNames findWinnersName(final CarGroup carGroup) {
+        return findCarNameWithWinningCar(carGroup, findWinningCar(carGroup));
     }
 
-    private MoveDistance findWinningScore(final CarGroups cars) {
-        return cars.getCarGroups().stream()
-            .map(Car::getMoveDistance)
-            .max(MoveDistance::compareTo)
+    private Car findWinningCar(final CarGroup carGroup) {
+        return carGroup.getCarGroup().stream()
+            .max(Car::compareTo)
             .orElseThrow(NoSuchElementException::new);
     }
 
-    private CarNames findCarNameWithWinningScore(final CarGroups cars, final MoveDistance moveDistance){
-        List<CarName> carNames = cars.getCarGroups().stream()
-            .filter(car -> car.getMoveDistance().equals(moveDistance))
+    private CarNames findCarNameWithWinningCar(final CarGroup carGroup, final Car winningCar){
+        List<CarName> carNames = carGroup.getCarGroup().stream()
+            .filter(car -> car.isSameMoveDistance(winningCar.getMoveDistance()))
```

Comment:
CarGroup 객체가 갖고 있는 상태를 꺼내지 말고 메세지를 보내는 것은 어떨까요? 🤔

```java
    public List<CarName> findCarNames(Car winningCar) {
        return this.carGroup.stream()
                .filter(car -> car.isSameMoveDistance(winningCar))
                .map(Car::getCarName)
                .collect(Collectors.toList());
    }
```

```suggestion
        List<CarName> carNames = carGroup.findCarNames(winningCar);
```

그리고 Car 는 Comparable 를 확장했고 거리로 비교를 하고있으니 쉽게 구현할 수 있을것 같아요 :)

Reply:
너무 좋은거 같아요
이 부분 피드백 반영해서 오늘 MR 올려놓을게요!

Reply:
커밋이 누락된것 같아요 😭 
확인부탁드립니다!

Reply:
하나가 누락되었네요 ㅠㅠ.. 다시 커밋했습니다